R/stanreg.R

```diff
@@ -73,6 +89,11 @@ stanreg <- function(object) {
     residuals <- ytmp - mu
   }
   names(eta) <- names(mu) <- names(residuals) <- ynames
+  if (family$family == "beta") {
+    eta_z <- linear_predictor(coefs_z, z, object$offset)
+    phi <- family_phi$linkinv(eta_z)
+    # residuals <- ytmp - rbeta(length(eta), mu * phi, (1 - mu) * phi) # this is not what betareg does
```

Comment:
what does betareg do? for `stan_glm` we don't do what `glm` does. we always to residuals on the outcome scale (check out the Details section of `?"stanreg-methods"`).